packages/cmd/src/profiles/CommandProfiles.ts

```diff
@@ -67,6 +131,8 @@ export class CommandProfiles {
         if (this.map.get(type) != null) {
             if (name != null && name.trim().length > 0) {
                 for (const prof of this.map.get(type)) {
+
+                    // TODO(Kelosky): is this in error if the name is not contained in IProfile?
```

Comment:
I like the `Kelosky TODOs`, but should we try to handle it (in case it is *in* error)?

Reply:
I added this comment because I think it was a but in the existing code base.  I didn't capture the steps to reproduce (which I should have).  I think perhaps these could be tracked as separate defects?